ops/model.py

```diff
@@ -1073,8 +1073,7 @@ def is_ready(self) -> '_ContainerReady':
          from the underlying Pebble operations will log error messages rather than
          raising exceptions.
 
-        Example:
-            ```
+        Example::
```

Comment:
I believe you need a blank line after the `Example::` line for it to render as a monospaced code block.